### PR TITLE
Update web_with_oc_server.adoc

### PR DESCRIPTION
**Pull request by jnweiger**
new richdocuments 390 and later is incompatible with web600 and before

### CODE REVIEW

File: modules/owncloud_web/pages/web_with_oc_server.adoc [hunk 1]
```diff
@@ -152,6 +152,7 @@ Make sure that ONLYOFFICE works as expected in the regular UI then add the follo
 ----
 
 NOTE: Adjust the URL in the example according to your setup.
+NOTE:
```
Comment (phil-davis) [inline modules/owncloud_web/pages/web_with_oc_server.adoc:155]: bonus note

Comment (EParzefall) [inline modules/owncloud_web/pages/web_with_oc_server.adoc:155]:
```suggestion
```

File: modules/owncloud_web/pages/web_with_oc_server.adoc [hunk 2]
```diff
@@ -176,6 +177,7 @@ Make sure that Collabora Online works as expected in the regular UI, then add th
 ----
 
 NOTE: Adjust the URL to match your setup.
+NOTE: richdocuments >= 3.9.0 requires web >= 7.0.0 (The configuration is silently ignored in web <= 6.0.0)
```
Comment (EParzefall) [inline modules/owncloud_web/pages/web_with_oc_server.adoc:180]:
```suggestion
NOTE: richdocuments >= 3.9.0 requires web >= 7.0.0. (The configuration is silently ignored in web <= 6.0.0)
```

**Review by EParzefall — APPROVED** [review]

File: modules/owncloud_web/pages/web_with_oc_server.adoc [hunk 3]
```diff
@@ -176,6 +176,7 @@ Make sure that Collabora Online works as expected in the regular UI, then add th
 ----
 
 NOTE: Adjust the URL to match your setup.
+NOTE: richdocuments >= 3.9.0 requires web >= 7.0.0. (The configuration is silently ignored in web <= 6.0.0)
```
Comment (EParzefall) [inline modules/owncloud_web/pages/web_with_oc_server.adoc:179]:
```suggestion
NOTE: richdocuments >= 3.9.0 requires web >= 7.0.0. (The configuration is silently ignored in web <= 6.0.0.)
```

**Pull request merged**